ibis/tests/expr/test_interactive.py

```diff
@@ -51,42 +51,42 @@ def test_repr_png_is_not_none_in_not_interactive(con):
 
 
 def test_default_limit(con):
-    table = con.table('functional_alltypes')
+    table = con.table('functional_alltypes').select("id", "bool_col")
 
     with config.option_context('interactive', True):
         repr(table)
 
     expected = """\
-SELECT *
+SELECT `id`, `bool_col`
```

Comment:
Any particular reason to change these tests?

Reply:
If a table is wide, we only compute the columns that are repr'd. The `functional_alltypes` table is wide, so it's truncated. I explicitly selected a small number of columns to try and avoid the test depending on the terminal width.

Reply:
Ah, got it!